src/gui/models/globalPotentialModel.cpp

```diff
@@ -0,0 +1,85 @@
+#include "gui/models/globalPotentialModel.h"
+#include "kernels/potentials/directional.h"
+GlobalPotentialModel::GlobalPotentialModel(const std::vector<std::unique_ptr<ExternalPotential>> &globalPotentials)
```

Comment:
Wooop, wooop! That's the sound of the (blank line) police!
```suggestion

GlobalPotentialModel::GlobalPotentialModel(const std::vector<std::unique_ptr<ExternalPotential>> &globalPotentials)
```

Reply:
Agh!